### PR TITLE
Addition of overview page

### PR DESCRIPTION
**Pull request by dboling19**
# Description

Addition of overview page - Testing database pushing and pulling.,

## Fixes 

Please list any issues that are addressed by this change

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Tested Locally

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: templates/overview.html.twig [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+  <style>
+      .w3-theme {
+          color:#fff !important; 
+          background-color:#40114f !important;
+      }
+      .disabled {
+          opacity: 0.6;
+      }
+  </style>
```
Comment (davidwkdavidwk) [inline templates/overview.html.twig:12]: Inline CSS is discouraged.  I'm going to let it slide here but I'm adding a new issue to implement better asset management to help with this kind of thing.

**Review by davidwkdavidwk — CHANGES_REQUESTED** [review]
This is a great start.  In addition to the comments I've already made I would like to see some tests to make sure that announcement creation is working as expected.  [https://symfony.com/doc/current/testing.html](https://symfony.com/doc/current/testing.html)

File: templates/overview.html.twig (hunk 1 above)
Comment (davidwkdavidwk) [reply templates/overview.html.twig:12]: Issue #16 created to address this

File: .env.local [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+###< symfony/mailer ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
+#
+# DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
+DATABASE_URL="mysql://dboling:SD90mac0601!@127.0.0.1:3306/communicator?serverVersion=5.7"
+# DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
+###< doctrine/doctrine-bundle ###
```
[the remark this answers is not in the document]
Comment (dboling19) [reply .env.local:32]: I thought I added that line already. I'll check again.

File: templates/overview.html.twig [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  <header>Communicator - Announcements</header>
```
[the remark this answers is not in the document]
Comment (dboling19) [reply templates/overview.html.twig:13]: I decided to just move this to a separate stylesheet.css file in the templates folder. This should allow me to customize a bit more CSS for the site(s) and keep things a bit neater overall.

File: src/Controller/OverviewController.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  /**
+   * @Route("/add", name="add_info")
+   */
+  public function add_info(): Response
+  {
+
+    $entityManager = $this->getDoctrine()->getManager();
+
+    $article = new Articles();
+    $articleId = strval($article->getId());
```
[the remark this answers is not in the document]
Comment (dboling19) [reply src/Controller/OverviewController.php:23]: You are right, but again I quickly tossed this in for demonstration purposes only to find that it didn't work after all. I'll just remove it for now, it won't be necessary for the final project at all.

Perhaps pulling the highest ID of the table might give me the results I was after, but ultimately it doesn't really matter in this case.

**Comment by davidwkdavidwk** [comment]
Closing this PR because it was superseded by #21

**Pull request closed without merging**